network-api/networkapi/wagtailpages/pagemodels/dear_internet.py

```diff
@@ -0,0 +1,61 @@
+from django.db import models
+
+from wagtail.admin.edit_handlers import FieldPanel, StreamFieldPanel
+from wagtail.core.models import Page
+from wagtail.core.fields import StreamField
+
+from . import customblocks
+from .mixin.foundation_metadata import FoundationMetadataPageMixin
+from ..utils import set_main_site_nav_information
+
+
+class DearInternetPage(FoundationMetadataPageMixin, Page):
+    intro_text_1 = models.CharField(
+        max_length=500,
+        help_text='Intro text 1',
+    )
+
+    intro_text_2 = models.CharField(
+        max_length=500,
+        help_text='Intro text 2',
+    )
+
+    intro_text_3 = models.CharField(
+        max_length=500,
+        help_text='Intro text 3',
+    )
+
+    intro_text_4 = models.CharField(
+        max_length=500,
+        help_text='Intro text 4',
+    )
+
+    letters = StreamField([
+        ('letters', customblocks.DearInternetLetterBlock()),
+    ])
+
+    cta = models.CharField(
+        max_length=500,
+    )
+
+    cta_button_text = models.CharField(
+        max_length=100,
+    )
+
+    cta_button_link = models.URLField()
+
+    content_panels = Page.content_panels + [
+        FieldPanel('intro_text_1'),
+        FieldPanel('intro_text_2'),
+        FieldPanel('intro_text_3'),
+        FieldPanel('intro_text_4'),
```

Comment:
this would then become a `StreamFieldPanel('intro_texts')`

Reply:
updated